libs/ui/src/screens/token/components/activity/activity.tsx

```diff
@@ -11,11 +13,16 @@ import {
 export const Activity: FC = () => {
   const {
     params: {
-      token: { tokenAddress }
+      token: { tokenAddress: basicTokenAddress }
```

Comment:
do not do so deep destructurization, it looks painful.  
just use `params.token.tokenAddress`